nsqd/lookup.go

```diff
@@ -54,10 +55,11 @@ func connectCallback(n *NSQD, hostname string) func(*lookupPeer) {
 		for _, topic := range n.topicMap {
 			topic.RLock()
 			if len(topic.channelMap) == 0 {
-				commands = append(commands, nsq.Register(topic.name, ""))
+				commands = append(commands, nsq.Register(topic.name, "", int(atomic.LoadInt32(&topic.paused))))
```

Comment:
As per my comment https://github.com/nsqio/go-nsq/pull/305#issuecomment-670968864, I think we should explore a new `nsqlookupd` protocol command like `SyncState` (or `UpdateRegistration`, naming TBD) that is sent whenever a topic/channel pause state changes.

Reply:
1. Add new command SyncState for topic/channel pause state change.
2.I think we can keep the ability of pause state delivery in command Register, through it we can reduce a call of command SyncState when topic/channel create.